src/__fixtures__/class-return-type.d.ts

```diff
@@ -4,6 +4,7 @@ declare namespace google {
      */
     namespace script {
         interface PublicEndpoints {
+            [key: string]: (...args: unknown[])=>unknown;
```

Comment:
```suggestion
            [key: string]: (...args: any[])=>any;
```

Reply:
Why change to `any`? 

`unknown` works and isn't going to cause issues with strict linters

https://www.typescriptlang.org/play?#code/C4TwDgpgBAYgrgOwMbAJYHsEFkCGYoC8UA3gFBQVQDaAHgFxQDOwATqggOYC6DAFAHSCcLDowY4EIKlwCUhAHxQJIANykAvqVLtgEFgDMcSaAAU4AIwA2qJAFEEAEzDodjEuUpUA1hBANmbJw8UAJCImJQiF4I6ADuCNJyBIpRMfEaWkiWOIxuAOIAggDKAMLWEAjAADwAKlAQNLqObvDIaJi4+ETE6oo9WggQsVCFpeWVVWZWNvZOLpWM8rwyQA

Reply:
dts-dom doesn't currently support `unknown`. will update